data/feeds.yml

```diff
@@ -251,7 +251,7 @@ Anoop C S:
 'Gluster on kshlm''s blog':
   image: kshlm.png
   rounded: true
-  feed: https://kshlm.in/tag/gluster/rss
+  feed: https://kshlm.in/index.xml
```

Comment:
The correct feed for posts tagged gluster is https://kshlm.in/tags/gluster/index.xml